scripts/skip-doc-change.sh

```diff
@@ -5,7 +5,7 @@ CHANGED_FILES=$(git diff --name-only "$TRAVIS_COMMIT_RANGE")
 
 skip=0
 #files to be skipped
-declare -a FILES=(^docs/ .md$ ^scripts/ LICENSE .travis.yml .mergify.yml .github .gitignore)
```

Comment:
why did we remove `.travis.yml` @Madhu-1 ?

Reply:
if I make some changes in Travis related to Kube, skipping will avoid testing it